roles/qemu-setup/defaults/main.yml

```diff
@@ -1,7 +1,8 @@
+---
 # Copyright (c) Stephen Bates, 2022
 
 # Do we update when we install packages
-qemu_setup_update_cache: no
+qemu_setup_update_cache: false
```

Comment:
I think we can remove this and just always update cache as long as state is present.